app/models/invenio_rdm_converter.rb

```diff
@@ -0,0 +1,48 @@
+include Sufia::Export
+
+# Convert a GenericFile including metadata, permissions and version metadata into a PORO
+# so that the metadata can be exported in json format using to_json
+#
+class InvenioRdmConverter < Sufia::Export::Converter
```

Comment:
So instead of using "GenericFile", we use this for invenioRDM record? Could we rename this to `InvenioRdmRecordConverter` maybe? Then do the same for "Collection", instead name the class `InvenioRdmCommunityConverter`.

Reply:
We can renname it , that makes sense to me. It's still technically using the GenericFile class from ActiveFedora (sufia?), but it's just storing it in a PORO that has different labels and functions that we want to use to make the conversions.